api/handlers/changes.js

```diff
@@ -449,7 +449,8 @@ var init = function(since) {
     since: since,
     heartbeat: true,
     feed: 'longpoll',
-    include_docs: true
+    include_docs: true,
+    return_docs: false,
```

Comment:
This is not a valid couchdb/nano parameter, so we don't need this change.

Reply:
I've reverted this change so I can backport to `2.14.x` today. Let me know if this was intentional...